### DISCUSSION TITLE
Add Astro user management implementation

### DISCUSSION THREAD

**Discussion by alexwhitmore** [supabase/supabase#19549] [Feature Requests]
# Improve documentation

## Link 

In the “Web App Tutorials” section.

## Describe the improvement

It would be awesome to add an [Astro](https://github.com/withastro/astro) implementation to the web app tutorials, providing one more user management web app example.

## Additional context

If there’s interest in adding this, I would love to take it on!

**Comment by charislam** [comment]
Would be great to have Astro! Feel free to get started on this if you're interested!
